backend/drm/drm-properties.h

```diff
@@ -56,4 +56,7 @@ bool wlr_drm_get_connector_props(int fd, uint32_t id, union wlr_drm_connector_pr
 bool wlr_drm_get_crtc_props(int fd, uint32_t id, union wlr_drm_crtc_props *out);
 bool wlr_drm_get_plane_props(int fd, uint32_t id, union wlr_drm_plane_props *out);
 
+bool wlr_drm_get_prop(int fd, uint32_t obj, uint32_t prop, uint64_t *ret);
+void *wlr_drm_get_prop_blob(int fd, uint32_t obj, uint32_t prop, size_t *ret_len);
```

Comment:
We still need the union wlr_drm_*_props, because they contain the prop argument. I would consider these all to be internal to the DRM backend and probably not particularly useful for a user, so I feel these don't need to be exposed.
Even if a user wanted to muck around with DRM properties for whatever reason, as long as we give them the DRM fd, they can just do it themselves.